jenkins/docker/Dockerfile

```diff
@@ -130,7 +130,7 @@ ENV MAVEN_REPO /var/maven_local_repository
 ENV MVN mvn --batch-mode
 
 RUN cd /tmp && \
-  git clone https://github.com/google/protobuf.git && \
+  git clone https://github.com/google/protobuf.git &&  \
```

Comment:
Does this really make a difference? How about changing "make -j4" to "make -j6"?